pdctl/command/region_command.go

```diff
@@ -191,7 +191,7 @@ func decodeProtobufText(text string) (string, error) {
 // NewRegionWithCheckCommand return a region with check subcommand of regionCmd
 func NewRegionWithCheckCommand() *cobra.Command {
 	r := &cobra.Command{
-		Use:   "check [miss-replica|extra-replica|down-replica|pending-replica|incorrect-ns]",
+		Use:   "check [miss-peer|extra-peer|down-peer|pending-peer|down-learner|pending-learner|incorrect-ns]",
```

Comment:
do we need using voter and learner here?

@disksing

Reply:
Currently we don't need to distinguish down voter and down learner since learner only exists in the intermediate process of add peer. But we will need to distinguish them later if we want to use learner in other ways.
For now both fine by me.